vas2nets/models.py

```diff
@@ -0,0 +1,13 @@
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class VoiceCall(models.Model):
+    created_at = models.DateTimeField()
+    msisdn = models.CharField(null=False, blank=False, max_length=36)
+    duration = models.IntegerField(null=False, blank=False)
+    reason = models.CharField(null=False, blank=False, max_length=36)
```

Comment:
What about having a `choices` attribute on here, since the reason field is limited to a specific set of choices?

Reply:
I avoided this incase they make changes.

Reply:
:+1: